auth/src/main/java/com/firebase/ui/auth/util/ui/FlowHolder.java

```diff
@@ -0,0 +1,24 @@
+package com.firebase.ui.auth.util.ui;
+
+import android.app.Application;
+import android.support.annotation.RestrictTo;
+
+import com.firebase.ui.auth.data.model.FlowParameters;
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+public class FlowHolder extends ViewModelBase<FlowParameters> {
+    private FlowParameters mFlowParams;
+
+    public FlowHolder(Application application) {
+        super(application);
+    }
+
+    @Override
+    protected void onCreate(FlowParameters args) {
+        mFlowParams = args;
+    }
+
+    public FlowParameters getParams() {
```

Comment:
nit: seems like this could be defined as `T getArguments()` in the `ViewModelBase` class?